code/LeftAndMain.php

```diff
@@ -171,9 +171,15 @@ class LeftAndMain extends Controller implements PermissionProvider
      * Current pageID for this request
      *
      * @var null
+     * @deprecated 2.4.0 use recordID instead.
      */
     protected $pageID = null;
 
+    /**
+     * ID of the current record for this request
+     */
+    protected ?int $recordID = null;
```

Comment:
Won't the fact we're suddenly no longer updating $pageID lead to it potentially breaking any subclasses that rely on it?

Reply:
Updated